src/main/java/unrc/dose/Comment.java

```diff
@@ -8,10 +13,96 @@
 * description     :varchar(300)    not null
 * challenge_id    :integer     not null
 * user_id         :integer    not null
-**/
-package unrc.dose;
+* comment_id      :integer
+*/
+public class Comment extends Model {
 
-import org.javalite.activejdbc.Model;
+  /**
+  *Create a new comment.
+  *@param title this is the comment's title
+  *@param description this the comment's body
+  *@param challengeId this is the challenge's id which is commented
+  *@param userId this is the user's id who commented
+  *@return the comment created
+  **/
+  public static Comment createComment(final String title,
+  final String description, final int challengeId, final int userId) {
+    if (description == null || title == null) {
+    throw new IllegalArgumentException("Comment and title can't be empty");
+    }
```

Comment:
There is a missing check for empty strings and there is an indentation missing (the throw statements should be indented)